ABJC/UI/Subviews/Episode Row & Card/EpisodeCardView.swift

```diff
@@ -15,13 +15,13 @@ extension LibraryView
         @EnvironmentObject var session: SessionStore
         
         /// Size
-        private var size: CGSize = CGSize(width: 548, height: 308.25 )
```

Comment:
Might be best to change back to these dimensions, mine are rather large

Reply:
Yep. you're right. I reverted that by accident. I changed it back in the latest commit ed89bcad4acea77a5ad1932d5704587e87a26b64